### ISSUE TITLE
JTextField background color overrides disabledBackground and inactiveBackground

### ISSUE THREAD

**Issue by FoxyBOA** [weisJ/darklaf#61]
Hello!

[As I see](https://github.com/weisJ/darklaf/blob/ece71b32bb0b4600d4dd98936ab8fd515f045196/core/src/main/resources/com/github/weisj/darklaf/properties/ui/text.properties#L38) the L&F defines pair disabledBackground/inactiveBackground color for most UI controls.

Unfortunately, in my examples background color never changes for readonly (editable=false) or disabled elements. **Is it a bug or expected behavior?**

E.g. see draft code below

```
        LafManager.install();
        
        System.out.println("TextField.inactiveBackground="+UIManager.getColor("TextField.inactiveBackground"));
        System.out.println("TextField.disabledBackground="+UIManager.getColor("TextField.disabledBackground"));

        String text = "";
        int columns = 20;
        JTextField textField = new JTextField(text, columns);
        
        JTextField textField1 = new JTextField(text, columns);
        textField1.setEditable(false);
//        textField1.setBackground(UIManager.getColor("TextField.inactiveBackground"));

        JTextField textField2 = new JTextField(text, columns);
        textField2.setEnabled(false);
//        textField2.setBackground(UIManager.getColor("TextField.disabledBackground"));

```

**Comment by weisJ** [comment]
Will be fixed with new version. Thank you for reporting!

**Comment by weisJ** [comment]
Fix available with `1.4.0.0`
